pkg/domain/infra/abi/containers.go

```diff
@@ -794,3 +794,17 @@ func (ic *ContainerEngine) ContainerCleanup(ctx context.Context, namesOrIds []st
 	}
 	return reports, nil
 }
+
+func (ic *ContainerEngine) ContainerInit(ctx context.Context, namesOrIds []string, options entities.ContainerInitOptions) ([]*entities.ContainerInitReport, error) {
+	var reports []*entities.ContainerInitReport
+	ctrs, err := getContainersByContext(options.All, options.Latest, namesOrIds, ic.Libpod)
```

Comment:
```suggestion
	ctrs, err := getContainersByContext(options.All, options.Latest, namesOrIds, ic.Libpod)
```
Just querying the containers causes them to be initialized?  I suspect a C&P error here.  I am also confused how you are going to create the container if a "not found" error is going to return.

Reply:
This is correct.  If the container input is bad (i.e. not found), then it returns and does not proceed.  This is the exact behavor of v1.